content/en/docs/reference/node/kubelet-checkpoint-api.md

```diff
@@ -25,6 +25,87 @@ should create the checkpoint archive to be only accessible by the `root` user. I
+     Kubelet Checkpoint API. Only authorized users or service accounts should have the 
+     necessary permissions to initiate checkpoint operations.
+
+   - Attribute-based access control (ABAC): ABAC allows access control decisions to be
+     based on attributes associated with the user, request, or other relevant factors.
+     Consider using ABAC policies to define fine-grained access rules for the Kubelet
+     Checkpoint API.
+
+   - Webhook authentication and authorization: Kubernetes supports webhook mechanisms for
+     authentication and authorization. You can integrate external authentication and
+     authorization systems by configuring webhooks to make access control decisions for the
+     Kubelet Checkpoint API.
+
+   - Network segmentation: Deploy the Kubernetes cluster in a network environment with proper
+     segmentation and firewall rules. Limiting access to the Kubelet's API endpoints reduces
+     the attack surface and protects against unauthorized access.
+
+4. **Secure storage of checkpoint archives**: Storing checkpoint archives securely is essential 
+   to prevent unauthorized access and tampering. Consider the following measures:
+
+   - Secure storage location: Store checkpoint archives in a secure directory with restricted 
+     access permissions. The underlying CRI implementation should ensure that the checkpoint 
+     archive is only accessible by the root user.
+
+   - Monitoring and auditing: Implement monitoring and auditing mechanisms to track access to 
+     the checkpoint archive storage location. This helps detect any unauthorized access attempts 
+     and provides an audit trail for accountability.
+
+5. **Secure deletion of checkpoint archives**: When checkpoint archives are no longer needed, 
+   securely delete them to prevent unauthorized recovery of sensitive data. Ensure that deletion 
+   processes comply with secure deletion standards and overwrite the data with random values to
+   make it unrecoverable.
+
+   By implementing these security measures, you can mitigate the risks associated with checkpointing 
+   containers and protect sensitive data from unauthorized access or exposure.
+
+6. **Integrity protection**:If the checkpoint includes sensitive data or data that requires protection against 
+   unauthorized modifications, integrity protection measures should be implemented. This typically involves using 
+   cryptographic mechanisms such as digital signatures or message authentication codes (MACs) to ensure the integrity of 
+   the checkpoint archive. These mechanisms verify that the checkpoint has not been tampered with during storage or 
+   transit.
+  
+7. **Determine sensitivity**:Before proceeding with integrity protection measures, it is essential to evaluate the 
+   sensitivity of the data within the container checkpoint. Confirm whether the checkpoint contains any sensitive or 
+   confidential information that needs to be protected.
+
```

Comment:
Point 1 is already solved by the implementation. Point 4 is more or less the same.

Point 5 is very vague. Not sure it is helpful.

The integrity protection parts are a good idea, but I think that it should happen automatically and not be left to the user.